beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -328,3 +326,15 @@ func (s *Service) hasBlockAndState(ctx context.Context, blockRoot [32]byte) bool
 	hasState := hasStateSummary || s.cfg.beaconDB.HasState(ctx, blockRoot)
 	return hasState && s.cfg.chain.HasBlock(ctx, blockRoot)
 }
+
+func seenAttCacheKey(slot primitives.Slot, committeeID primitives.CommitteeIndex, aggregationBits []byte) string {
+	totalLen := 8 + 8 + len(aggregationBits)
```

Comment:
It might be helpful to add an explanation of what these numbers represent for anyone unfamiliar with the code. Otherwise they just seem like magic numbers